clang/test/CodeGen/2005-01-02-ConstantInits.c

```diff
@@ -7,13 +7,6 @@
 struct X { int a[2]; };
 extern int bar();
 
-//.
-// CHECK: @test.i23 = internal global i32 4, align 4
-// CHECK: @i = global i32 4, align 4
-// CHECK: @Arr = global [100 x i32] zeroinitializer, align 16
-// CHECK: @foo2.X = internal global ptr getelementptr (i8, ptr @Arr, i64 196), align 8
-// CHECK: @foo2.i23 = internal global i32 0, align 4
```

Comment:
Something went wrong with update_cc_test_checks here? It has `--check-globals`, so this was supposed to be checked...

Reply:
Hmm good point - I think the issue is this file uses `--global-value-regex "@.+"`, which prevents matching because the update script internals actually prefix `"^@"` to the regex (and obviously `"^@@.+"` doesn't match)
https://github.com/llvm/llvm-project/blob/0cff3e85db00b5f425cc4ed0d6921445afa891ca/llvm/utils/UpdateTestChecks/common.py#L2280-L2281

I'm not sure what's best to do here - we can remove `--global-value-regex "@.+"`, which restores CHECK lines for these globals, but that will also add CHECK lines for some attributes because they match the regex on line 2281 there...

We could add `--global-value-regex "[A-Za-z].*"` to capture globals only, not sure if that's too hacky?

Reply:
Interesting. So the way this is designed you can't cleanly distinguish globals and attributes.

I think you solution sounds fine.

Reply:
Yeah, possibly something that could be addressed in the script if desired - not sure how much it comes up.

Ok great, I've updated the `--global-value-regex` argument and fixed up the missing global CHECKs.